README.fr.md

```diff
@@ -2808,6 +2809,41 @@ Avec les options :
 -x # active le mutex lock (off par défaut)
 ```
 
+### Multi-threading
+
+_Cette partie est basée sur [une réponse StackOverflow][so-answer] par Konstantin._
+
+Sinatra n'impose pas de modèle de la concurrence. Sinatra est thread-safe, vous pouvez
```

Comment:
I guess it should be something like "Sinatra n'impose pas de modèle pour faire de la concurrence" or "Sinatra n'impose pas de modèle concurrent".